src/aggregate-deployment-planning.rst

```diff
@@ -88,43 +106,86 @@ You may be able to run a full study within these activity thresholds provided yo
 
 Otherwise, you will need to set up a billing account with Google.
 
+.. note::
+
+  These usage limits are well within what is needed to try out ODK,
+  to determine if it would be a good fit for your data collection needs.
+  
+  Because of this, and its ease of setup,
+  we recommend using Google App Engine 
+  in test deployments and pilot studies,
+  even if you will likely need a custom deployment later on.
+  
+.. _aggregate-deployment-open-source:
+
 Open source
 ~~~~~~~~~~~~~~~
 
-The ODK software is open source and available for use without charge. It is important to recognize that the open source software model does place additional responsibilities on the users of that software.
+The ODK software is `free <https://www.gnu.org/philosophy/free-sw.en.html>`_, `open source <https://opensource.com/resources/what-open-source>`_, and available for use without charge. 
+
+It is important to recognize that the open source software model does place additional responsibilities on the users of that software.
 
 Unless you pay for assistance when technical support is needed, you will be required to take the initiative to research and find answers, and to perform technical support tasks yourself. 
 
-And, unless you contribute back to the open source community through your involvement in the user groups and through contributions to the project, this software will become irrelevant and obsolete.
+Finally, unless you and others contribute back to Open Data Kit through involvement in the community and contributions to the project, this software will become irrelevant and obsolete.
+
+.. seealso:: `Learn more about participating in ODK <https://opendatakit.org/participate/>`_
 
-App Engine is usually sufficient
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _aggregate-deployment-app-engine-sufficient:
 
-For most users, Google App Engine will be the easiest and most cost-effective option. The http://opendatakit.appspot.com site, which is lightly used, typically costs just over the minimum charge of $2.10/week unless there are a lot of users that have set up publishing actions (we purge these periodically). 
+.. App Engine is usually sufficient
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Why is this commented out?

Reply:
see #525 
This info seemed possibly out of date, so I tried to confirm it.
I couldn't find anything in Google App Engine's current pricing info about weekly billing or weekly minimums.